### PR TITLE
[Merged by Bors] - chore(RingTheory/MvPolynomial/Homogeneous): extract api lemma from existing proof

### PR DESCRIPTION
**Pull request by jcommelin**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RingTheory/MvPolynomial/Homogeneous.lean [hunk 1]
```diff
@@ -261,18 +261,19 @@ theorem sub (hφ : IsHomogeneous φ n) (hψ : IsHomogeneous ψ n) : IsHomogeneou
 
 end CommRing
 
+lemma totalDegree_le (hφ : IsHomogeneous φ n) : φ.totalDegree ≤ n := by
```
Comment (ocfnash) [inline Mathlib/RingTheory/MvPolynomial/Homogeneous.lean:264]:
```suggestion
/-- The homogeneous degree bounds the total degree.

See also `MvPolynomial.IsHomogeneous.totalDegree` when `φ` is non-zero. -/
lemma totalDegree_le_of_isHomogeneous (hφ : IsHomogeneous φ n) : φ.totalDegree ≤ n := by
```

**Review by ocfnash — APPROVED** [review]
Thanks!

bors d+

File: Mathlib/RingTheory/MvPolynomial/Homogeneous.lean [hunk 2]
```diff
@@ -261,18 +261,22 @@ theorem sub (hφ : IsHomogeneous φ n) (hψ : IsHomogeneous ψ n) : IsHomogeneou
 
 end CommRing
 
+/-- The homogeneous degree bounds the total degree.
+
+See also `MvPolynomial.IsHomogeneous.totalDegree` when `φ` is non-zero. -/
+lemma totalDegree_le_of_isHomogeneous (hφ : IsHomogeneous φ n) : φ.totalDegree ≤ n := by
```
Comment (jcommelin) [inline Mathlib/RingTheory/MvPolynomial/Homogeneous.lean:267]: Whoops, I didn't notice that you also proposed a namechange. But we're in the `IsHomogeneous` namespace. So I think we should go with the short name that allows for dot-notation. Do you agree?
```suggestion
lemma totalDegree_le (hφ : IsHomogeneous φ n) : φ.totalDegree ≤ n := by
```

Comment (ocfnash) [reply Mathlib/RingTheory/MvPolynomial/Homogeneous.lean:267]: Yes, I prefer your name!

**Pull request closed without merging**